lib/commands/cryptographic-identities.ts

```diff
@@ -605,12 +605,12 @@ export class ImportCryptographicIdentity implements ICommand {
 					this.$logger.error(err.message + os.EOL + "Verify that you have provided the correct file and password and try again.");
 				}
 
-				if (noErrorOccurred) {
+				if (noErrorOccurred || importType === CryptographicIdentityConstants.X509CERTIFICATE) {
```

Comment:
I'm confused... If something's wrong, noErrorOccurred will be false, but we'll still break the for and a messages "Successfully imported.... " will be shown to the user. What am I missing here?